### PR TITLE
[Concept Entry] C# Classes 

### PR DESCRIPTION
**Pull request by narcissus123**
### Description

- Added classes concept for C# and an image for the partial classes subsection.
- Closes Issue #2758

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/classes/classes.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+In C#, a **class** serves as a blueprint or template for creating objects. It plays a fundamental role in defining the structure, behavior, and attributes of these objects. A class can be thought of as a user-defined data type that encapsulates both data (attributes) and the methods (functions) that operate on that data.
+
+## Syntax for Declaring Classes
+
+To declare a class in C#, you use the `class` keyword, followed by the class name. Class names should follow C# naming conventions (typically using PascalCase). The class definition is enclosed within curly braces {}.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/classes/classes.md:22]:
```suggestion
To declare a class in C#, use the `class` keyword, followed by the class name. Class names should follow C# naming conventions (typically using PascalCase). The class definition is enclosed within curly braces {}.
```

File: content/c-sharp/concepts/classes/classes.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+Static classes are defined using the `static` keyword and exclusively contain static members, such as methods, properties, and fields. Unlike regular classes, static classes cannot be instantiated with the `new` keyword. Instead, their members are accessed using the class name itself. These classes are commonly used for utility functions or to group related functionality.
+
+## Partial classes
+
+Partial classes in C# enable class definitions to be split across multiple files. Each part of the class is defined in a separate file and combined at compile time to create a single class. This is valuable for scenarios where a class becomes too large or complex, or when multiple developers need to work on different aspects of the class simultaneously.
+
+![Partial Classes](https://raw.githubusercontent.com/Codecademy/docs/main/media/c-sharp-partial-class.png)
+
+```cs
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/classes/classes.md:101]: Have a sentence introducing this code block and explaining its relationship to the image.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@narcissus123 I did a first review and left a couple of comments.

**Comment by narcissus123** [comment]
@SSwiniarski, thank you for your initial review. I have applied the changes. If you have any additional feedback or require further adjustments, please let me know.

**Review by SSwiniarski — APPROVED** [review]
@narcissus123 Looks good for a second review.

**Comment by narcissus123** [comment]
@SSwiniarski, may I ask how long it takes for my pull request to go through a second review? I'm really looking forward to getting my entry in for Hacktoberfest.

**Review by caupolicandiaz — APPROVED** [review]
@narcissus123 @SSwiniarski Overall, looks good. Just made a few minor language and formatting edits. This is ready to merge

**Comment by narcissus123** [comment]
@SSwiniarski @caupolicandiaz Thank you so much for reviewing and accepting my pull request.

**Pull request merged**